diesel/src/pg/types/record.rs

```diff
@@ -190,4 +221,34 @@ mod tests {
             .get_result(&conn);
         assert_eq!(Ok(true), res);
     }
+
+    #[test]
+    fn serializing_named_composite_types() {
```

Comment:
I think this test case should live in `diesel_tests/tests/custom_types.rs` because we link that file everywhere.